### PR TITLE
Add a news post for the 10th anniversary

### PR DESCRIPTION
**Pull request by abcdefg30**
Sorry for being late.

fullpage

### CODE REVIEW

**Comment by pchote** [comment]
I recently outlined some of the other improvements at https://ppmforums.com/viewtopic.php?t=43447 which you could include here.

**Comment by abcdefg30** [comment]
Added a bit more information.

**Comment by abcdefg30** [comment]
Removed the image as requested on IRC:
fullpage

**Comment by abcdefg30** [comment]
Updated again.

File: content/news/10th-anniversary.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+</div>
+
+<hr>
+
+Meanwhile, we are working towards a new playtest. Some core improvements and additions are:
+<ul>
+<li>A new unit control bar that provides mouse and touch accessibility.</li>
+<li>Unit stances that are easier to use and behave more like the original games.</li>
+<li>Overhauled structure hit boxes, with weapons, engineers, etc. accounting for their physical size instead of acting only at their centre.</li>
```
Comment (cjshmyr) [inline content/news/10th-anniversary.md]: Commas made this read weird to me at first. Could use paul's original formatting?

> Overhauled structure hit boxes, with weapons/engineers/etc. accounting for ...

Comment (abcdefg30) [inline content/news/10th-anniversary.md]: Ok, updated.

**Pull request merged**